contracts/QuestChain.sol

```diff
@@ -17,118 +16,34 @@ contract QuestChain is
     IQuestChain,
     ReentrancyGuard,
     Initializable,
-    AccessControl,
-    Pausable
+    Pausable,
+    AccessControl
 {
     bytes32 public constant OWNER_ROLE = bytes32(0);
```

Comment:
Instead of `OWNER_ROLE` can we make sure of `DEFAULT_ADMIN_ROLE`? I may be wrong.

Reply:
Yeah good catch.